index.html

```diff
@@ -166,8 +166,8 @@
 
         // Draw Age Row Highlighters
         let row_height = tree.height / 4;
-        draw.rect(tree.width, tree.height / 4).attr({fill: '#4d361740'}).click(hideHelp);
-        draw.rect(tree.width, tree.height / 4).attr({fill: '#4d361740'}).click(hideHelp).y(row_height * 2);
+        draw.rect(tree.width, tree.height / 4).attr({fill: '#4d3617', opacity:"0.3"}).click(hideHelp);
+        draw.rect(tree.width, tree.height / 4).attr({fill: '#4d3617', opacity:"0.3"}).click(hideHelp).y(row_height * 2);
```

Comment:
I guess both `"0.3"` and `0.3` work here, but at all the other places you have chosen so use the number instead of the number string.